pkg/datapath/iptables/ipset/ipset.go

```diff
@@ -50,17 +51,13 @@ type Initializer interface {
 }
 
 type initializer struct {
-	cond    *sync.Cond
-	counter *int
+	once sync.Once
+	wg   *lock.StoppableWaitGroup
```

Comment:
Should be moved to the `iptables/ipset: Add initializer to delay entries deletion` commit. The same applies for the other changes in this file.

Reply:
Oops, I've messed up the rebase. I'm gonna fix it.

Reply:
Thanks!